release-notes/1.0/1.0.1-sdk-release-notes.md

```diff
@@ -6,15 +6,15 @@
 * [.NET Core 1.0.4](https://github.com/dotnet/core/blob/master/release-notes/1.0/1.0.4.md)
 * [.NET Core 1.1.1](https://github.com/dotnet/core/blob/master/release-notes/1.1/1.1.1.md)
 
-[.NET Core 1.0.4](https://github.com/dotnet/core/blob/master/release-notes/1.0/1.0.4.md) was released today. Both releases include reliability updates to improve the quality of .NET Core. You can download the .NET Core Runtime releases via our [.NET Core Runtimes download page](https://www.microsoft.com/net/download/core#/runtime). If you are looking for the .NET Core SDK, to get the latest tools, try the [.NET Core SDK download page](https://www.microsoft.com/net/download/core#/sdk).
+[.NET Core 1.0.4](https://github.com/dotnet/core/blob/master/release-notes/1.0/1.0.4.md) was released today. Both releases include reliability updates to improve the quality of .NET Core. You can download the .NET Core Runtime releases via our [.NET Core Runtimes download page](https://dotnet.microsoft.com/download/dotnet-core/current/runtime). If you are looking for the .NET Core SDK, to get the latest tools, try the [.NET Core SDK download page](https://dotnet.microsoft.com/download).
```

Comment:
dotnet-core/current/runtime only offers Windows and Linux. Is that intentional?

Reply:
I think it was done that way intentionally in the past but we're adding macOS to that page on a PR I'm working on